### PR TITLE
Add util to find role

### PR DESCRIPTION
**Pull request by scottbucher**

### CODE REVIEW

File: src/utils/client-utils.ts [hunk 1]
```diff
@@ -64,6 +65,26 @@ export class ClientUtils {
...
     }
 
+    public static async findRole(guild: Guild, input: string): Promise<Role> {
+        try {
+            let discordId = RegexUtils.discordId(input);
+            if (discordId) {
+                return await guild.roles.fetch(discordId);
+            }
+
+            return guild.roles.cache.find(r => r.name.toLowerCase().includes(input));
```
Comment (KevinNovak) [inline src/utils/client-utils.ts:75]: Could you rename "r" to "role"?

Comment (scottbucher) [reply src/utils/client-utils.ts:75]: ;-; fine

Comment (KevinNovak) [reply src/utils/client-utils.ts:75]: Also input should be lowercased as well

File: src/utils/client-utils.ts [hunk 2]
```diff
@@ -64,6 +65,26 @@ export class ClientUtils {
...
     }
 
+    public static async findRole(guild: Guild, input: string): Promise<Role> {
+        try {
+            let discordId = RegexUtils.discordId(input);
+            if (discordId) {
+                return await guild.roles.fetch(discordId);
+            }
+
+            return guild.roles.cache.find(role => role.name.toLowerCase().includes(input));
```
Comment (KevinNovak) [inline src/utils/client-utils.ts]: Could you make this `(await guild.roles.fetch()).find(`

**Pull request merged**